models/recurly__churn_analysis.sql

```diff
@@ -31,7 +31,6 @@ subscription_churn_reason as (
             when lower(expiration_reason) = 'nonpayment_gift' then 'gift ended'
             when lower(expiration_reason) = 'nonpayment' then 'non-payment'
             when lower(expiration_reason) = 'non renewing' then 'non-renewing'
--- haven't confirmed the below, will revisit with production data
```

Comment:
Let's remove this for now, we can still come back to this. I just don't want to leave a comment inline upon release.

Reply:
Can do!